app/assets/stylesheets/darkswarm/_shop-popovers.css.scss

```diff
@@ -25,54 +24,27 @@
 
   .joyride-nub.right {
     top: 38px;
-    border-color: #999 !important;
+    border-color: $grey-800 !important;
     border-top-color: transparent !important;
     border-right-color: transparent !important;
     border-bottom-color: transparent !important;
   }
 
-  .progress {
-    background-color: #13bf85;
-    padding: 0;
-    border: none;
-    color: white;
-    font-size: 0.75rem;
-    font-style: oblique;
-    line-height: 1;
-    height: auto;
-
-    .right {
-      padding: 0.5rem 0.25rem 0 0;
-    }
-
-    .meter {
-      background-color: #0b8c61;
-      padding: 0.5rem 0.25rem;
-      border-right: 1px solid #539f92;
-    }
+  ul, li {
+    list-style: none;
+    margin: 0;
+    font-size: 0.875rem;
   }
 
-  .expanded {
-    ul, li {
-      list-style: none;
-      margin: 0;
-      font-size: 0.875rem;
-    }
-
-    li {
-      background-color: #13bf85;
-      padding: 0 0.25rem;
-      margin-bottom: 2px;
-      color: white;
-    }
-
-    li.cost {
-      background-color: #0b8c61;
-    }
+  li {
+    border-bottom: .1em solid $grey-700;
```

Comment:
isn't probably 1px what you want? I can't imagine a border scaling according to the font size. I would look broken, I guess.

Reply:
I usually prefer `em` over `px`. It doesn't look odd at all. Actually, browser make `1px` bigger than one pixel when zooming in so that it doesn't look odd.

No zoom:
![Screenshot from 2020-05-07 16-00-59](https://user-images.githubusercontent.com/3524483/81260048-8fbb9080-907c-11ea-991b-8d4e69bef30f.png)

With zoom:
![Screenshot from 2020-05-07 16-01-50](https://user-images.githubusercontent.com/3524483/81260061-96e29e80-907c-11ea-8b51-b32b602b7849.png)

I think it's good scale everything and keep relative proportions. When an element becomes bigger, it needs a bigger border as well.

Reply:
well, I'm not so sure. This is the sort of question we need to ask the designer.

Reply:
@yukoosawa Do you prefer border width defined in `px` or `em` or something else? Does it depend on the case or is there a general rule?